cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -343,6 +343,10 @@ def get_volume_stats(self, refresh=False):
             self._stats = self._get_volume_stats()
         return self._stats
 
+    def _get_connection_capabilities(self):
+        return ['vmware_service_instance_uuid:%s' %
+                self.session.vim.service_content.about.instanceUuid]
```

Comment:
This capability needs to be included by Nova driver as well, in the connector details.